### ISSUE TITLE
bug(a11y): Calendar: Current day not marked with `aria-current="date"`

### ISSUE THREAD

**Issue by kyubisation** [lyne-design-system/lyne-components#1881]
Feedback from Mürsel

The current day is written in bold and is therefore visually recognizable, but is not recognized and output as such by the screen reader.

It should have `aria-current="date"` set on it.
